--- tracker/udp/conn-client.go.orig	2022-05-16 05:10:10.000000000 +0300
+++ tracker/udp/conn-client.go	2022-06-01 09:34:27.000000000 +0300
@@ -38,9 +38,9 @@
 			// TODO: Do bad things to the dispatcher, and incoming calls to the client if we have a
 			// read error.
 			cc.readErr = err
-			if !cc.closed {
-				panic(err)
-			}
+//			if !cc.closed {
+//				panic(err)
+//			}
 			break
 		}
 		err = cc.d.Dispatch(b[:n], addr)
